src/backend/distributed/executor/multi_utility.c

```diff
@@ -296,9 +304,9 @@ multi_ProcessUtility(Node *parsetree,
 									  "move all tables.")));
 		}
 	}
-	else if (!schemaNode)
+	else
```

Comment:
Isn't the else block entirely superfluous now? I.e. shouldn't the contained if be one layer up?